src/seq_mv/csr_matop_device.c

```diff
@@ -1807,15 +1827,24 @@ hypre_CSRMatrixTransposeDevice(hypre_CSRMatrix  *A,
    }
    else
    {
+      if ( !hypre_HandleSpTransUseVendor(hypre_handle()) )
```

Comment:
Do we want to add an option to set this analogous to `HYPRE_SetSpMVUseVendor`?

Reply:
Yeah, but for selecting which transpose implementation to use. I see you added `HYPRE_SetSpMVUseVendor`, but nothing like `HYPRE_SetSpTransUseVendor`.

Reply:
Oh. Sorry. I misunderstood the question (I was thinking about SpMV only). I can add that as well. Thanks!

Reply:
`HYPRE_SetSpTransUseVendor` has been added.